Geometry/fieldFactory_func.f90

```diff
@@ -0,0 +1,80 @@
+!!
+!! Module to build new fields. Called by geometryReg_mod
+!!
```

Comment:
I think it is now the other way around, right? 
This module calls `geomReg` and adds the geometry.